client/allocrunner/taskrunner/task_dir_hook.go

```diff
@@ -93,11 +100,10 @@ func setEnvvars(envBuilder *taskenv.Builder, fsi drivers.FSIsolation, taskDir *a
 	// Set the host environment variables for non-image based drivers
 	if fsi != drivers.FSIsolationImage {
 		// COMPAT(1.0) using inclusive language, blacklist is kept for backward compatibility.
-		denylist := conf.ReadAlternativeDefault(
+		filter := strings.Split(conf.ReadAlternativeDefault(
```

Comment:
just making this consistent with it's clone in plugins:
https://github.com/hashicorp/nomad/blob/master/plugins/drivers/testutils/testing.go#L252